.changeset/wild-ligers-cross.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit-labs/task': patch
```

Comment:
If this is actually a no-op change, it shouldn't need a release, right?

Reply:
yeup, you're correct. sounds good!

Reply:
i think we commented in the same cache frame lol